engines/mohawk/riven_stacks/domespit.cpp

```diff
@@ -125,9 +125,25 @@ void DomeSpit::checkSliderCursorChange(uint16 startHotspot) {
 }
 
 int16 DomeSpit::getSliderSlotAtPos(uint16 startHotspot, const Common::Point &pos) const {
+	// Emperically found min x and max x hotspot are used to bound mouse position into
+	// the slider area vertically. This and the y mouse position being directly put into
+	// the slider area allows the user to move the cursor out of the slider area and still
+	// be able to move the slider.
+	int16 minXHotspot = 211; // suitable min x value hotspot for all domes
+	int16 maxXHotspot = 407; // suitable max x value hotspot for all domes
+
+	// Now got find out which slider is being moved
```

Comment:
Please fix / remove that comment. This function gets the slider slot closest to `pos`, which is not necessarily the slider being moved.